File: Interview_question/python/Problem2.py
def nonDivisibleSubset(S):
    List=[]
    Num=1
    while Num<=S:
        num=int(input("enter any number"))
        List.append(num)
        Num+=1
    print(List)
    num1 = int(input("enter any number "))
    i=0
    C=0
    while i<len(List):
        j=i
        while j<len(List):
            if (List[i]+List[j])%num1==0:
                C+=1
            j+=1
        i+=1
    print(C)
S=int(input("enter any number"))
nonDivisibleSubset(S)

File: Interview_question/JavaScript/Problem2.js
function nonDivisibleSubset (){
    const number = [1,7,2,4];
    let S= " ";
    let arr = [];
    for (let i= 0; i< number.length; i++){
        for (let j =0; j<number.length; ++j){
            if (number[i]!=number[j]){
                S=number[i]+number[j];
                if (S%3!=0){
                    arr.push(number[i])
                }
            }
        }
        arr = arr.filter((v,i,a) => a.indexOf (v) === i);
    }
    console.log(arr)
}
nonDivisibleSubset ();
